atlasdb-cassandra/src/test/java/com/palantir/atlasdb/cassandra/CassandraReloadableKeyValueServiceRuntimeConfigTest.java

```diff
@@ -107,11 +108,12 @@ public void fetchBatchCountAlwaysUsesRuntimeConfigValue() {
 
     @Test
     public void sweepReadThreadsAlwaysUsesRuntimeConfigValue() {
```

Comment:
there's one place where the install is still used here as well

Reply:
See above